radix-engine-tests/benches/metadata_validation.rs

```diff
@@ -0,0 +1,20 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use radix_common::prelude::*;
+use radix_engine::object_modules::metadata::MetadataValueValidationError;
+use scrypto::prelude::CheckedUrl;
+use std::hint::black_box;
+
+#[allow(unused_must_use)]
+fn bench_validate_urls(c: &mut Criterion) {
+    c.bench_function("metadata_validation::validate_urls", |b| {
+        b.iter(|| {
+            black_box(
+                CheckedUrl::of("https://www.example.com/test?q=x")
+                    .ok_or(MetadataValueValidationError::InvalidURL("".to_owned())),
```

Comment:
MINOR: This is always, valid, right? So we should probably just `unwrap()` here - otherwise greedy allocation of a string might slow down the test a little.

Reply:
Happy to unwrap.